src/components/link/AbstractLink.component.jsx

```diff
@@ -45,41 +45,41 @@ const AbstractLink = React.createClass({
 			nextProps.target !== this.props.target ||
 			nextProps.targetHandlePosition !== this.props.targetHandlePosition;
 	},
-	renderLinkSourcehandle(){
-		if(this.props.linkSourceHandleComponent){
-			return <LinkHandle
+	renderLinkSourcehandle() {
+		if (this.props.linkSourceHandleComponent) {
+			return (<LinkHandle
 				component={this.props.linkSourceHandleComponent}
 				onDrag={this.props.onSourceDrag} onDragEnd={this.props.onSourceDragEnd}
-				position={this.props.sourceHandlePosition || this.props.source.position}
-			/>
+				position={this.props.sourceHandlePosition || this.props.source.getIn(['graphicalAttributes', 'position'])}
```

Comment:
im going to do this on another PR https://github.com/Talend/react-flow-designer/issues/192